src/data/PostsData.ts

```diff
@@ -0,0 +1,94 @@
+import { IPost } from "typings/flights"
+
+const PostsData:IPost[] = [
+    {
+        id: 0,
+        thumbnailId: 0,
+        title: "Nowa data startu",
+        slug: "nowa-data",
+        content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam dapibus dapibus consequat. Suspendisse non nibh pellentesque, venenatis neque a, faucibus mi. Sed sed nisi non massa sodales fermentum. Etiam pulvinar eleifend tortor in mattis. Vivamus sit amet pretium massa. Aenean nisi lectus, accumsan eu pretium sit amet, tristique quis urna. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam dapibus dapibus consequat. Suspendisse non nibh pellentesque, venenatis neque a, faucibus mi. Sed sed nisi non massa sodales fermentum. Etiam pulvinar eleifend tortor in mattis. Vivamus sit amet pretium massa. Aenean nisi lectus, accumsan eu pretium sit amet, tristique quis urna.",
+        emailOfAuthor: "kiko.loureiro@gmail.com",
+        publishedAt: "03/04/2022 15:42",
```

Comment:
```suggestion
        publishedAt: "2022-11-18T12:45:37.979Z",
```
This is the format returned by backend